src/promptflow/promptflow/_cli/data/package_tool/tool.py.jinja2

```diff
@@ -0,0 +1,11 @@
+from promptflow import tool
+from promptflow.connections import CustomConnection
+
+
+@tool(name="{{ tool_name }}", description="This is {{ tool_name }} tool")
+def {{ tool_name }}(connection: CustomConnection, input_text: str) -> str:
+    # Replace with your tool code.
```

Comment:
So after the init, user needs to manually replace the code here? I thought we can generate it for user.